pkg/formatted/color.go

```diff
@@ -16,19 +16,64 @@ package formatted
 
 import (
 	"io"
+	"sync"
 
 	"github.com/fatih/color"
 )
 
+var (
+	mux       sync.Mutex
```

Comment:
mux should be instance variable and not package var since it only locks the `rc.cached` and not a package level variable.